templates/tftmpl/integration_test.go

```diff
@@ -109,6 +109,8 @@ func TestRenderTFVarsTmpl(t *testing.T) {
 		registerAPI2    bool
 		registerAPISrv2 bool
 		registerWeb     bool
+		registerKV      bool
+		registerKVMulti bool
```

Comment:
Actually I can just get rid of these two conditions and just always add the KV, then it wouldn't create so many extra paths in the test code, and the checking should still work... it would even make sure that `recurse true` is the only one that picks up the services in the `registerKVMulti `block

Reply:
never mind, can't do that right now because of the spacing issue. Leaving this one as is